packages/components/src/internal/components/ontology/OntologyTreeSearchContainer.tsx

```diff
@@ -1,25 +1,81 @@
-import React, { ChangeEvent, FC, memo, MouseEvent, useCallback, useEffect, useMemo, useState } from 'react';
+import React, { ChangeEvent, FC, memo, MouseEvent, ReactNode, useCallback, useEffect, useMemo, useState } from 'react';
 
 import { Alert, searchUsingIndex } from '../../..';
 
 import { ConceptModel, OntologyModel, PathModel } from './models';
-import { fetchAlternatePaths } from './actions';
+import { fetchAlternatePaths, getOntologyDetails } from './actions';
 
 const CONCEPT_CATEGORY = 'concept';
 const SEARCH_LIMIT = 20;
 
+interface OntologySearchInputProps {
+    ontologyId: string;
+    initCode?: string; //initial concept code
+    inputName: string;
+    className: string;
+    searchPathClickHandler: (code: string) => void;
+    showBrowserHandler?: () => void;
+    fieldName?: string;
+}
+
+export const OntologySearchInput: FC<OntologySearchInputProps> = memo(props => {
+    const {ontologyId, searchPathClickHandler, ...rest} = props;
+    const [ontologyModel, setOntologyModel] = useState<OntologyModel>();
+    const [error, setError] = useState<string>();
+
+    useEffect(() => {
+        if (ontologyId) {
+            getOntologyDetails(ontologyId)
+                .then((ontology: OntologyModel) => {
```

Comment:
nit: The `ontology` parameter doesn't need to have it's type redeclared here as `getOntologyDetails` already specifies the return type.